nucleus/common/internal-api/src/main/java/fish/payara/internal/notification/PayaraNotifierConfiguration.java

```diff
@@ -56,7 +56,7 @@ public interface PayaraNotifierConfiguration extends ConfigBeanProxy {
 
     String DEFAULT_ENABLED_VALUE = "false";
     String DEFAULT_NOISY_VALUE = "true";
-    String DEFAULT_EVENT_FILTER = "WARNING";
+    String DEFAULT_EVENT_FILTER = "WARNING_(DEFAULT)"; // A safe dummy value used for the upgrade service.
```

Comment:
I'm not sure how I feel about this name. It's a little misleading, as it's not really the default?
The get command also now _expects_ to encounter an exception with this default.

I don't think I'm a fan of this strategy. I shall mull on it over the weekend.

Reply:
See https://github.com/payara/Payara-Enterprise/pull/1655#discussion_r1810438628